config/astral.yml

```diff
@@ -19,9 +19,25 @@ shared:
   audit_log_file: <%= "#{Rails.root.join('log')}/astral-audit.log" %>
 
 test:
+  oidc_provider:
+    issuer: "http://oidc_provider:8300/v1/identity/oidc/provider/astral"
+  initial_user:
+    name: "test"
+    password: "test"
+    email: "test@example.com"
   cert_ttl: <%= 24.hours.in_seconds %>
 
 development:
+  oidc_provider:
+    issuer: "http://oidc_provider:8300/v1/identity/oidc/provider/astral"
+  initial_user:
+    name: "test"
+    password: "test"
+    email: "test@example.com"
 
 production:
   vault_create_root: false
+  oidc_provider:
```

Comment:
I think the empty defaults should go in the 'shared' section. Also, I like the nested Yaml, but we currently cannot override nested keys using ENV vars, so maybe we should keep them flat for now with a prefix suggesting the area of concern, eg "oidc_issuer"

Reply:
or, alternatively -- the dev and test config could go in 'shared' and prod will set to empty values as here

Reply:
made them flat